CMakeLists.txt

```diff
@@ -74,6 +74,67 @@ if ((NOT "${RELEASE_TYPE}" STREQUAL "PRODUCTION") AND (NOT "${RELEASE_TYPE}" STR
     set(RELEASE_TYPE "DEV")
 endif()
 
+# VIRCADIA_OPTIMIZE
+# Variable determining vircadia optimization. If not set, it defaults to true.
+# It's used to determine build flags for main codebase and for VCPKG dependencies.
+# Should be set to false to get completely unoptimized build for easier line-by-line debugging
+
+if( NOT WIN32 )
+    if(NOT DEFINED VIRCADIA_OPTIMIZE)
+        message("Enabling code optimization for Vircadia and compiled dependencies")
+        set(VIRCADIA_OPTIMIZE true CACHE BOOL "Enable code optimization for Vircadia and compiled dependencies")
+    endif()
+
+    #compiler needs to be detected before building VCPKG dependencies
+    set(CMAKE_PLATFORM_INFO_DIR "${CMAKE_CURRENT_BINARY_DIR}")
+    include(CMakeDetermineCXXCompiler)
+
+    set(VIRCADIA_OPTIMIZE_FLAGS "")
+
+            set(VIRCADIA_OPTIMIZE_FLAGS "-O3 -fPIC")
```

Comment:
Is this in the wrong line or something? I would assume that it should be part of the Clang if statement instead of just floating around here.

Reply:
Thanks! That line was left there by mistake. I removed it in most recent commit.